packages/xrpl/src/models/methods/ledger.ts

```diff
@@ -67,6 +67,112 @@ export interface LedgerRequest extends BaseRequest, LookupByLedgerRequest {
   queue?: boolean
 }
 
+/**
+ * Retrieve information about the public ledger. Expects a response in the form
+ * of a {@link LedgerResponseExpanded}. Will return full JSON-formatted transaction data instead of string hashes.
+ *
```

Comment:
Probably should be a follow up issue, but I think it'd be worth linking to the xrpl.org docs pages for these requests since not only do they go into more depth on how the requests work, they also have a "Try It" button that let's you see the request in action using the websocket tool :)

Ex. https://xrpl.org/ledger.html#ledger